lib/features/home_view/widgets/recetly_active_section.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:catchqr/theme/app_theme.dart';
+import 'package:catchqr/utils/context_extensions.dart';
+import 'package:flutter/material.dart';
+
+class RecentlyActiveWidget extends StatelessWidget {
+  const RecentlyActiveWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          context.localize.recently_active,
+          style: context.textTheme.headline,
+        ),
+        const SizedBox(height: 15),
+        Expanded(
+          child: ListView.separated(
+            itemCount: 5,
+            separatorBuilder: (context, index) => const SizedBox(height: 8),
+            itemBuilder: (context, index) {
+              return RecentlyActiveGameWidget(
+                gameName: 'Gra $index',
+                onTap: () {
+                  // Navigate to the game
+                },
+              );
+          }),
+        ),
+      ],
+    );
+  }
+}
+
+class RecentlyActiveGameWidget extends StatelessWidget {
+  const RecentlyActiveGameWidget({
+    super.key, 
+    required this.gameName,
+    required this.onTap
+  });
+
+  final String gameName;
+  final Function() onTap;
```

Comment:
I encourage to always use full functional types with return type as well:

```dart
 final void Function() onTap;
```

I believe for this one, there's a shortcut alias type:
```dart
final VoidCallback onTap;
```

Reply:
also this callback can probably be nullable as it doesn't make us any difference to accept null as well?